Issue when trying to run with VueFirebase

Hi 

I am getting the following error whenever I add 'vuetify-nuxt-module' with 'nuxt-vuefire'.

Without 'vuetify-nuxt-module'  my web application runs but soon as I add vuetify module I get the following errors : 

Any ideas ?

[nuxt] [request error] [unhandled] [500] Cannot find module './call-credentials'
Require stack:
- /com.docker.devenvironments.code/nuxt-vuetify/@grpc/grpc-js
  at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)  
  at Module._load (node:internal/modules/cjs/loader:920:27)  
  at Module.require (node:internal/modules/cjs/loader:1141:19)  
  at require (node:internal/modules/cjs/helpers:110:18)  
  at ./@grpc/grpc-js:20:28  
  at ViteNodeRunner.directRequest (./node_modules/vite-node/dist/client.mjs:341:11)  
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)  
  at async ViteNodeRunner.cachedRequest (./node_modules/vite-node/dist/client.mjs:197:14)  
  at async ViteNodeRunner.dependencyRequest (./node_modules/vite-node/dist/client.mjs:231:12)  
  at async ./@firebase/firestore:8:31

--- a/playground/nuxt.config.mts
+++ b/playground/nuxt.config.mts
@@ -5,7 +5,7 @@ import LayerModule from './layer-module'
 
 export default defineNuxtConfig({
   extends: ['layers/vuetify-layer'],
-  ssr: true,
+  // ssr: false,
   sourcemap: true,
   imports: {
     autoImport: true,

--- a/src/utils/configure-nuxt.ts
+++ b/src/utils/configure-nuxt.ts
@@ -31,12 +31,10 @@ export function configureNuxt(
   else if (typeof styles === 'object' && typeof styles?.configFile === 'string')
     nuxt.options.css.unshift(styles.configFile)
 
-  if (includeTransformAssetsUrls) {
-    if (typeof nuxt.options.vite.vue?.template?.transformAssetUrls === 'undefined') {
-      nuxt.options.vite.vue ??= {}
-      nuxt.options.vite.vue.template ??= {}
-      nuxt.options.vite.vue.template.transformAssetUrls = transformAssetUrls
-    }
+  if (includeTransformAssetsUrls && typeof nuxt.options.vite.vue?.template?.transformAssetUrls === 'undefined') {
+    nuxt.options.vite.vue ??= {}
+    nuxt.options.vite.vue.template ??= {}
+    nuxt.options.vite.vue.template.transformAssetUrls = transformAssetUrls
   }
 
   extendWebpackConfig(() => {

--- a/src/utils/configure-nuxt.ts
+++ b/src/utils/configure-nuxt.ts
@@ -18,8 +18,10 @@ export function configureNuxt(
 
   const runtimeDir = ctx.resolver.resolve('./runtime')
 
-  nuxt.options.build.transpile.push(configKey)
-  nuxt.options.build.transpile.push(runtimeDir)
+  if (!nuxt.options.ssr) {
+    nuxt.options.build.transpile.push(configKey)
+    nuxt.options.build.transpile.push(runtimeDir)
+  }
 
   nuxt.options.css ??= []
   if (typeof styles === 'string' && ['sass', 'expose'].includes(styles))
@@ -30,10 +32,11 @@ export function configureNuxt(
     nuxt.options.css.unshift(styles.configFile)
 
   if (includeTransformAssetsUrls) {
-    nuxt.options.vite.vue ??= {}
-    nuxt.options.vite.vue.template ??= {}
-    if (typeof nuxt.options.vite.vue.template.transformAssetUrls === 'undefined')
+    if (typeof nuxt.options.vite.vue?.template?.transformAssetUrls === 'undefined') {
+      nuxt.options.vite.vue ??= {}
+      nuxt.options.vite.vue.template ??= {}
       nuxt.options.vite.vue.template.transformAssetUrls = transformAssetUrls
+    }
   }
 
   extendWebpackConfig(() => {
@@ -99,16 +102,9 @@ export function configureNuxt(
     })
   }
 
-  if (nuxt.options.dev) {
+  if (nuxt.options.dev || ctx.dateAdapter) {
     addPlugin({
       src: ctx.resolver.resolve(runtimeDir, 'plugins/vuetify-date'),
     })
-  }
-  else {
-    if (ctx.dateAdapter) {
-      addPlugin({
-        src: ctx.resolver.resolve(runtimeDir, 'plugins/vuetify-date'),
-      })
-    }
   }
 }

--- a/src/utils/configure-vite.ts
+++ b/src/utils/configure-vite.ts
@@ -17,11 +17,18 @@ export function configureVite(configKey: string, nuxt: Nuxt, ctx: VuetifyNuxtCon
 
     viteInlineConfig.optimizeDeps = defu(viteInlineConfig.optimizeDeps, { exclude: ['vuetify'] })
 
-    viteInlineConfig.ssr ||= {}
-    viteInlineConfig.ssr.noExternal = [
-      ...(Array.isArray(viteInlineConfig.ssr.noExternal) ? viteInlineConfig.ssr.noExternal : []),
-      configKey,
-    ]
+    if (nuxt.options.ssr) {
+      viteInlineConfig.ssr ||= {}
+      viteInlineConfig.ssr.noExternal = [
+        ...(Array.isArray(viteInlineConfig.ssr.noExternal)
+          ? viteInlineConfig.ssr.noExternal
+          : viteInlineConfig.ssr.noExternal && typeof viteInlineConfig.ssr.noExternal !== 'boolean'
+            ? [viteInlineConfig.ssr.noExternal]
+            : []
+        ),
+        configKey,
+      ]
+    }
 
     viteInlineConfig.plugins.push(vuetify({ styles: true, autoImport: true }))
     viteInlineConfig.plugins.push(vuetifyStylesPlugin({ styles: ctx.moduleOptions.styles }, ctx.logger))

